json_templates.rdl_json_exposure1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "properties": {
+    "identifier": "https://datacatalog.worldbank.org/search/dataset/0064252/Central-Asia-exposure-dataset---Transport",
+    "title": "Central Asia Exposure Dataset - Transport",
+    "description": "Regional layer of transportation infrastructure in Central Asia (road, rail, bridges).",
+    "risk_data_type": "exposure",
+    "publisher": {
+        "name": "World Bank",
+        "email": "",
+        "url": "www.worldbank.org"
+        },
+    "version": 1,
+    "purpose": "Regional climate and disaster risk analysis for risk financing and risk management",
+    "project": "Strengthening Financial Resilience and Accelerating Risk Reduction in Central Asia - Regional risk assessment.",
+    "details": ""
+    "referenced_by": "",
+    "spatial": {
+      "countries": ["KGZ", "KAZ", "UZB", "TJK", "TKM"],
+      "gazetteerEntries": [],
```

Comment:
Like fields that contain only whitespace, empty arrays should be omitted from the data.